.ruby-version

```diff
@@ -0,0 +1 @@
+2.6.0
```

Comment:
Actually, should this be 2.6.3? Is there a reason for not using the latest version?

Reply:
@alyssais  Ruby 2.6.3 looks for bundler version >2.0. We can change bundler version in gemspec. https://github.com/fac/hermod/blob/master/hermod.gemspec#L28

I'm not sure why Hermod uses older version of bundler.

Reply:
upgraded to bundler 2 and ruby 2.6.3.